spec/routing/cost_types_routing_spec.rb

```diff
@@ -42,8 +42,12 @@
                                                        :action => 'set_rate',
                                                        :id => '5') }
 
-    it { expect(put('/cost_types/5/toggle_delete')).to route_to(:controller => 'cost_types',
-                                                            :action => 'toggle_delete',
-                                                            :id => '5') }
+    it { expect(delete('/cost_types/5')).to route_to(controller: 'cost_types',
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.